model/project_vars.go

```diff
@@ -216,15 +220,41 @@ func (projectVars *ProjectVars) GetRestrictedVars() map[string]string {
 	return restrictedVars
 }
 
-func (projectVars *ProjectVars) GetAdminOnlyVars() map[string]string {
+func (projectVars *ProjectVars) GetAdminOnlyVars(t *task.Task) (map[string]string, error) {
 	adminOnlyVars := map[string]string{}
-
-	for k, v := range projectVars.Vars {
-		if projectVars.AdminOnlyVars[k] {
-			adminOnlyVars[k] = v
+	if evergreen.IsSystemActivator(t.ActivatedBy) {
+		for k, v := range projectVars.Vars {
+			if projectVars.AdminOnlyVars[k] {
+				adminOnlyVars[k] = v
+			}
+		}
+	} else {
+		var u *user.DBUser
+		var err error
+		if t.ActivatedBy != "" {
+			u, err = user.FindOneById(t.ActivatedBy)
+		}
+		if err != nil {
```

Comment:
Le's not error in this case. We can just not give them permissions and log the error.

Reply:
gripped error because i wasnt super sure on how to log it

Reply:
grip.Error(message.WrapError(err, message.Fields{
"message": ...,
"task_id": ...
}))

Reply:
done